junebug/channel.py

```diff
@@ -142,3 +152,15 @@ def status(self):
         # TODO: Implement channel status
         status['status'] = {}
         return status
+
+    @inlineCallbacks
+    def send_message(self, amq_client, to_addr, content, **kw):
+        message = TransportUserMessage.send(
+            to_addr=to_addr, content=content, transport_name=self.id)
+        queue = '%s.outbound' % self.id
+        msg = yield amq_client.publish_message(message, routing_key=queue)
+        ret = {}
+        for key, val in msg.payload.iteritems():
+            if key in allowed_message_fields:
+                ret[key] = val
```

Comment:
Ok, now that I'm not reading this wrong, do we maybe want to add validation for the message fields given to `send_message`? To disallow fields that we will be replacing anyways, or fields that people shouldn't be setting. Or maybe that's something we should leave up to the api?